conda_forge_tick/make_graph.py

```diff
@@ -231,7 +230,8 @@ def _build_graph_sequential(
     mark_not_archived=False,
 ) -> None:
     for name in names:
-        if RNG.random() >= RANDOM_FRAC_TO_UPDATE:
+        if RNG.random() >= FRAC_MAKE_GRAPH:
+            logger.debug(f"skipping {name} due to random fraction to update")
```

Comment:
```suggestion
            logger.debug("skipping %s due to random fraction to update", name)
```

Reply:
I agree this is the better style (and have applied your patch) - I think we should include the ruff rule that enforces it. Can do this in a separate PR!